.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+	"typescript.tsdk": "node_modules\\typescript\\lib",
```

Comment:
This will need to be "meteor\\node_modules\\typescript\\lib"
Also should the slashes be \\ or / ? We should make sure to test on both windows and linux to make sure it is happy.